scss/_progressive-font.scss

```diff
@@ -0,0 +1,84 @@
+$_o-typography-progressive-font-fallbacks: (
+	sans: (
+		font: $o-typography-sans,
+		fallback: sans-serif,
+		fallback-scale: 0.87
+	),
+	sansBold: (
+		font: $o-typography-sans,
+		fallback: sans-serif,
+		fallback-scale: 0.83
+	),
+	sansData: (
+		font: $o-typography-sans,
+		fallback: sans-serif,
+		fallback-scale: 0.87
+	),
+	sansDataBold: (
+		font: $o-typography-sans,
+		fallback: sans-serif,
+		fallback-scale: 0.83
+	),
+	sansDataItalic: (
+		font: $o-typography-sans,
+		fallback: sans-serif,
+		fallback-scale: 0.87
+	),
+	serif: (
+		font: $o-typography-serif,
+		fallback: serif,
+		fallback-scale: 0.84
+	),
+	serifBold: (
+		font: $o-typography-serif,
+		fallback: serif,
+		fallback-scale: 0.84
+	),
+	serifItalic: (
+		font: $o-typography-serif,
+		fallback: serif,
+		fallback-scale: 0.84
+	),
+	serifDisplay: (
+		font: $o-typography-serif-display,
+		fallback: serif,
+		fallback-scale: 0.9
+	),
+	serifDisplayBold: (
+		font: $o-typography-serif-display,
+		fallback: serif,
+		fallback-scale: 0.9
+	),
+	serifDisplayItalic: (
+		font: $o-typography-serif-display,
+		fallback: serif,
+		fallback-scale: 0.9
```

Comment:
This is based on me running my chrome on my mac. There's obviously no way one number will work for all situations, but it gets it in the same ballpark as the webfont.